### DISCUSSION TITLE
Threading values?

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#215] [Q&A] [answered]
Ive yet to use the threading function on the H4 yet as I normally use tap and dies, but I have some sizes in between the tap and dies that I need to cut so I thought id single point thread cut them using the automatic thread feature.

But it seems to require some values that are confusing to me.

I understand the right endstop is where it will begin
the left endstop will be where it ends
The bottom endstop is the depth beginning
And the forward endstop is the depth ending

Is that correct?

Basic Thread Depth: The thread depth 
d is approximately 0.613 * pitch.

For an M11x1mm thread, the pitch is 1.0 mm.

So, the thread depth 
d would be:

d=0.613×Pitch
d=0.613×1.0mm
d≈0.613mm

Therefore, the radial infeed depth would be:

Infeed Depth
= 0.613 / 2 ≈ 0.307
 
So I would start at X-5.5 for the bottom endstop and the forward endstop would be 0.307 inwards from this 5.5 start point?

I wonder if it would be useful to have a built in table of common threads that one can cycle through and just click play where they only need to set the right and left endstops.

**Comment by duebelpapst** [comment, marked as answer]
Hi,

no the infeed is 0.613mm.
![Gewinde](https://github.com/user-attachments/assets/f75784ce-7d4f-4f31-b994-58c4468abc6c)

Regards,
Stefan

**Reply by Festivejelly** [reply]
Thanks

**Reply by Festivejelly** [reply]
I found this chart so could be useful to others:

https://i.accu.co.uk/documents/marketing/Thread-Size-Sheet-V2.jpg

![Thread-Size-Sheet-V2](https://github.com/user-attachments/assets/be5a31f5-6a07-41c5-bf9e-0c96e27fd576)
